Fix running multiple migration process in parallel (#5329)
element-hq/matrix-authentication-service: docs/development/database.md

```

Note that migrations are embedded in the final binary and can be run from the service CLI tool.

### Removing migrations

For various reasons, we may want to delete migrations.

> In case we do, we *must* declare that migration version as it is fine to be missing.

Reviewer: ```suggestion
In case we do, we *must* declare that migration version as allowed to be missing.
```
